### DISCUSSION TITLE
I wanted to comment on a resolved issue

### DISCUSSION THREAD

**Discussion by jrmoreno1** [dotnet/aspnetcore#53839] [General]
https://github.com/dotnet/aspnetcore/issues/45783 

Was "resolved" as basically "wont fix" because the person reporting it didn't have an acceptable use case.  I have the same request, but for a different reason, and wanted to provide it but couldn't.

Basically, I am creating an editcontext outside of the EditForm and wish to test to see if it is or would be valid.  I don't know what it would need a service provider for, I am using basic required and regular expression validation via annotation.

**Comment by gfoidl** [comment]
@jrmoreno1 in that case please open a new issue and add a link to the old (closed) one. Thanks!
